extensions/build.gradle

```diff
@@ -2,7 +2,7 @@ description = "Sparkling Water Extensions"
 
 dependencies {
   api(project(':sparkling-water-utils'))
-  api("org.scala-lang:scala-library:${scalaVersion}")
+  compileOnly("org.scala-lang:scala-library:${scalaVersion}")
```

Comment:
@mn-mikke Don't we need to keep Scala library in this case? Extensions are passed to H2O and it does not provide Scala library

Reply:
Yes, you're right but assembly extensions does that already. https://github.com/h2oai/sparkling-water/blob/master/assembly-extensions/build.gradle#L8

Or am I missing something?

Reply:
Good catch, all good then :)